packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor.cs

```diff
@@ -68,6 +68,7 @@ public void AddTableRows(int numberOfRowsToAdd)
 
                 tableData.Add(new SimpleTableRecord(
                     rowCountString,
+                    tableData.Count > 4 ? ((tableData.Count + i) % 4).ToString(CultureInfo.CurrentCulture) : null,
```

Comment:
If the other suggestion was accepted, then:
```suggestion
                    tableData.Count >= 4 ? (tableData.Count % 4).ToString(CultureInfo.CurrentCulture) : null,
```